data/config-schema/src/main/resources/net/corda/schema/configuration/db/1.0/corda.db.json

```diff
@@ -4,5 +4,56 @@
   "title": "Corda Database Configuration Schema",
   "description": "Configuration schema for the database subsection.",
```

Comment:
We should add a note here that this configuration cannot be updated dynamically through the REST endpoint.
Maybe we should even block somebody trying to push this config to the HTTP endpoint? @mnesbit, what do you think?

Reply:
added note